src/ol/renderer/webgl/Layer.js

```diff
@@ -81,6 +82,14 @@ class WebGLLayerRenderer extends LayerRenderer {
   getShaderCompileErrors() {
     return this.helper.getShaderCompileErrors();
   }
+
+  /**
+   * @inheritDoc
+   */
+  hasFeatureAtCoordinate(coordinate, frameState) {
```

Comment:
I think this function is never called: `ol/renderer/Map` has a `hasFeatureAtCoordinate` function that loops over all the renderers and calls `forEachFeatureAtCoordinate`.

Reply:
What about: https://github.com/fredj/openlayers/commit/3bf05265d5951fa21f56e7621a594807716b61e0 ?

Reply:
Nice catch! I'm adding your commit to the branch, and also remove the implementation in the base layer renderer. Thanks a lot